scriptmodules/emulators/hypseus.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="hypseus"
+rp_module_desc="Hypseus Singe - Laserdisc Emulator"
+rp_module_help="ROM Extension: .daphne\n\nCopy your Daphne roms to $romdir/daphne"
+rp_module_licence="GPL3 https://raw.githubusercontent.com/DirtBagXon/hypseus-singe/master/LICENSE"
+rp_module_repo="git https://github.com/DirtBagXon/hypseus-singe.git RetroPie"
+rp_module_section="exp"
+rp_module_flags="sdl2"
+
+function depends_hypseus() {
+    getDepends libvorbis-dev libogg-dev zlib1g-dev libmpeg2-4-dev libsdl2-dev libsdl2-image-dev libsdl2-ttf-dev cmake
+}
+
+function sources_hypseus() {
+    gitPullOrClone
+}
+
+function build_hypseus() {
+    local params=()
+    mkdir build
+    cd build
+    cmake ../src
+    make
+    cp hypseus ../hypseus.bin
+    md_ret_require="hypseus"
+}
+
+function install_hypseus() {
+    md_ret_files=(
+        'sound'
+        'pics'
+        'fonts'
+        'hypseus.bin'
+        'LICENSE'
+    )
+}
+
+function configure_hypseus() {
+    mkRomDir "daphne"
+    mkRomDir "daphne/roms"
+
+    addEmulator 0 "$md_id" "daphne" "$md_inst/hypseus.sh %ROM%"
+    addSystem "daphne"
+
+    [[ "$md_mode" == "remove" ]] && return
+
+    if [[ ! -f "$md_conf_root/daphne/hypinput.ini" ]]; then
+        cp -v "$md_data/hypinput.ini" "$md_conf_root/daphne/hypinput.ini"
+    fi
+    ln -snf "$romdir/daphne/roms" "$md_inst/roms"
+    ln -snf "$romdir/daphne/roms" "$md_inst/singe"
+    ln -sf "$md_conf_root/daphne/hypinput.ini" "$md_inst/hypinput.ini"
+
+    cat >"$md_inst/hypseus.sh" <<_EOF_
+#!/bin/bash
+dir="\$1"
+name="\${dir##*/}"
+name="\${name%.*}"
+
+if [[ -f "\$dir/\$name.commands" ]]; then
+    params=\$(<"\$dir/\$name.commands")
+fi
+
+if [[ -f "\$dir/\$name.singe" ]]; then
+    "$md_inst/hypseus.bin" singe vldp -retropath -framefile "\$dir/\$name.txt" -script "\$dir/\$name.singe" -homedir "$md_inst" -fullscreen \$params
+else
+    "$md_inst/hypseus.bin" "\$name" vldp -framefile "\$dir/\$name.txt" -homedir "$md_inst" -fullscreen \$params
+fi
+_EOF_
+    chmod +x "$md_inst/hypseus.sh"
+
+    chown -R $user:$user "$md_inst"
+    chown -R $user:$user "$md_conf_root/daphne/hypinput.ini"
```

Comment:
`-R` doesn't make sense on a single file. But this line won't be needed if using copyDefaultConfig